python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1743,6 +1754,10 @@ def stats(self, all: bool = False) -> List[ContainerStats]:
             "--no-trunc",
         ]
         full_cmd.add_flag("--all", all)
+
+        if None not in containers:
+            full_cmd += containers
```

Comment:
```suggestion
        if containers == []:
            return []
        elif containers is None:
            # the user didn't provide any filters
            pass
        else:
            full_cmd += to_list(containers)
```
with this the intent is easier to understand